crates/avalanche-types/src/key/secp256k1/txs/mod.rs

```diff
@@ -342,6 +342,7 @@ fn test_sort_output_owners() {
 /// ref. <https://pkg.go.dev/github.com/ava-labs/avalanchego/vms/secp256k1fx#Input>
 #[derive(Debug, Serialize, Deserialize, Eq, Clone, Default)]
 pub struct Input {
+    #[serde(rename = "signatureIndices")]
```

Comment:
Sorry, forgot to mention that you would also need to rename the field to `signature_indices`.

Reply:
If you don't want to do this, I will still accept the PR

Reply:
> Nit: Generally, it's probably better to do `$[serde(rename_all = "camelCase")]` so that you don't have to add this to every new field.

Yeah, I've been considering that, but it doesn't quite work for some fields like `node_id`. Still, may be better to just add it.